apps/dashboard/src/lib/components/Org/Settings/Profile.svelte

```diff
@@ -13,21 +13,29 @@
   import { t } from '$lib/utils/functions/translations';
   import LanguagePicker from './LanguagePicker.svelte';
   import { handleLocaleChange } from '$lib/utils/functions/translations';
-  import { updateProfileEmailValidation } from '$lib/utils/functions/validator';
+  import {
+    updateProfileEmailValidation,
+    updateProfileFullnameValidation,
+    updateProfileUsernameValidation
+  } from '$lib/utils/functions/validator';
 
   let avatar = '';
   let loading = false;
   let hasLangChanged = false;
   let locale = '';
 
   let errors = {
-    email: ''
-  }
+    email: '',
+    username: '',
+    fullname: ''
+  };
 
   async function handleUpdate() {
-    errors = updateProfileEmailValidation($profile.email)
+    errors.email = updateProfileEmailValidation($profile.email).email || '';
+    errors.username = updateProfileUsernameValidation($profile.username).username || '';
+    errors.fullname = updateProfileFullnameValidation($profile.fullname).fullname || '';
```

Comment:
You shouldn't create a validation for each field.

Please look at how we do validation here and if possible reuse it.

https://github.com/rotimi-best/classroomio/blob/main/apps/dashboard/src/routes/onboarding/%2Bpage.svelte#L126